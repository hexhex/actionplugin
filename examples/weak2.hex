a(y).
a(z).
f(X) v g(X) :- a(X).
:~ f(X). [1:1]
:~ g(X). [2:1]
