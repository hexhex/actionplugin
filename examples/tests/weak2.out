{f(z), f(y), a(z), a(y)}
Cost ([Weight:Level]): <[2:1]>
{g(z), f(y), a(z), a(y)}
Cost ([Weight:Level]): <[3:1]>
{g(y), f(z), a(z), a(y)}
Cost ([Weight:Level]): <[3:1]>
{g(z), g(y), a(z), a(y)}
Cost ([Weight:Level]): <[4:1]>
